packages/paste-core/components/detail-text/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@twilio-paste/detail-text",
+  "version": "0.0.0",
+  "category": "typography",
+  "status": "production",
+  "description": "Detail text is typography used for small, short pieces of secondary text content.",
```

Comment:
nit but I feel like `small, short` is redundant

Reply:
I agree, I got this from the spec: https://docs.google.com/document/d/19w39npHJgJkiLLQO9j3TnHrkrtOG_j25cbPJH0w3GB8/edit?pli=1#heading=h.gwj72pv8f47o

I can change it tho